dash_table/package.json

```diff
@@ -77,6 +78,9 @@
     "webpack-dev-server": "^3.1.5",
     "webpack-preprocessor": "^0.1.12"
   },
+  "files": [
+    "/dist/bundle.js"
```

Comment:
this is the only thing beside the md files, license and package.json that we want to publish